frontend/pages/admin/UserManagementPage/UserManagementPage.tsx

```diff
@@ -241,14 +245,17 @@ const UserManagementPage = (): JSX.Element => {
           toggleCreateUserModal();
           refetchInvites();
         })
-        .catch((userErrors: IApiError) => {
-          if (userErrors.errors?.[0].reason.includes("already exists")) {
-            dispatch(
-              renderFlash(
-                "error",
-                "A user with this email address already exists."
-              )
-            );
+        .catch((userErrors: { data: IApiError }) => {
+          if (userErrors.data.errors[0].reason.includes("already exists")) {
+            setCreateUserErrors({
+              email: "A user with this email address already exists",
+            });
+          } else if (
+            userErrors.data.errors[0].reason.includes("already invited")
+          ) {
+            setCreateUserErrors({
+              email: "A user with this email address has already been invited",
+            });
```

Comment:
Does the API return different status codes for these cases? If so, let's try using those. If it returns the same status code for both cases, that would be an appropriate time to parse the response object to determine which case it is.